packages/fxa-profile-server/lib/server/worker.js

```diff
@@ -29,7 +29,7 @@ exports.create = async function () {
       },
     },
   });
-  server.validator(require('@hapi/joi'));
+  server.validator(require('joi'));
```

Comment:
I'm not familiar w/ the code, but do we need to re-`require()` joi here? Or can we use the `Joi` instance we already required up from L6 above? (although I don't suspect it's a huge deal w/ Node's require caching, etc).